packages/extension/src/libs/RPChProvider/index.ts

```diff
@@ -0,0 +1,59 @@
+import RPChSDK, { JRPC, type Ops } from "@rpch/sdk";
+import { AbstractProvider } from "web3-eth/node_modules/web3-core"; // I had to import from web3-eth dependencies as web3-core has other version
+
+const FORCE_ZERO_HOP = true; // TODO: Change to false after integration for better privacy
+
+export const getSupportedRpchProvider = (
```

Comment:
This should be a static function returning the singleton SDK.

From the usages seems like the SDK could be instantiated multiple times.